packages/update-monitor/src/projects/optimism/contracts/getStateCommitmentChain.ts

```diff
@@ -12,18 +13,27 @@ export async function getStateCommitmentChain(
     provider,
   )
 
+  const addressManager: string = await stateCommitmentChain.libAddressManager()
+
+  const bondManager = await getCallResultWithRevert<string>(
+    provider,
+    addressManager,
+    'function getAddress(string implementationName) view returns(address)',
+    ['BondManager'],
+  )
```

Comment:
I'm going to approve, but why have this here instead of it being on the addressManager?